src/collectdctl.c

```diff
@@ -150,7 +150,7 @@ static int parse_identifier (lcc_connection_t *c,
     const char *value, lcc_identifier_t *ident)
 {
   char hostname[1024];
-  char ident_str[1024] = "";
+  char ident_str[1028] = "";
```

Comment:
Does this go into fixing "the format truncation warnings"?

Reply:
Yeah. The error was "output 2 or more bytes (assuming 1025) into a destination of size 1024" for line 170 below, so anything over 2 bytes would have probably worked.